xwwp.el

```diff
@@ -161,6 +161,71 @@ Interactively, URL defaults to the string looking like a url around point."
       (progn (xwidget-webkit-goto-url url)
              (switch-to-buffer-other-window (xwidget-buffer (xwidget-webkit-current-session)))))))
 
+;;; Adapted from EWW code to provide a DWIM style XWWP command
+(defcustom xwwp-search-prefix "https://google.com/search?q="
+  "Prefix URL to search engine."
+  :group 'xwwp
+  :type 'string)
+
+(defun xwwp--dwim-expand-url (url)
```

Comment:
I don't use the `--` convention for private function anywhere. Could you use `xwwp-dwim-expand-url` here instead ?